src/web/modules/agreements/list.njk

```diff
@@ -0,0 +1,130 @@
+{% from "agreements/status.macro.njk" import status %}
+
+{% extends "layout/layout.njk" %}
+
+{% set isTestData = account and not (account.type === "live") %}
+
+{% set linkQueryParams %}{% if service %}&account={{ accountId }}{% endif %}{% if filters.reference %}&reference={{ filters.reference }}{% endif %}{% endset %}
+
+{% block main %}
+  <span class="govuk-caption-m">
+    {% if service %}
+      <span>{{ service.name }}</span>
+    {% else %}
+      <span>GOV.UK Pay platform</span>
+    {% endif %}
+  </span>
+
+  <h1 class="govuk-heading-m">Agreements</h1>
+
+  <div class="transactions-filter__container govuk-body">
+    {% for status in ["created", "active", "cancelled", "all"] %}
+      <a
+        class="transactions-filter__item {% if selectedStatus === status %}selected{% endif %} no-decoration"
+        href="/agreements?status={{ status }}{{ linkQueryParams }}">
+        <span>{{ status | capitalize }}</span>
+      </a>
+    {% endfor %}
+  </div>
+
+  {% if filters.email or filters.reference or filters.gateway_payout_id %}
+    <div class="govuk-body">
+      <h4 class="govuk-heading-s">Filtered by</h4>
+      {% if filters.reference %}
+        <div class="govuk-body">
+          <div class="govuk-grid-column-one-third">Reference: </div>
+          <span><strong class="govuk-tag">{{ filters.reference }}</strong></span>
+        </div>
+      {% endif %}
+      {% if filters.email %}
+        <div class="govuk-body">
+          <div class="govuk-grid-column-one-third">Email:</div>
+          <span><strong class="govuk-tag">{{ filters.email }}</strong></span>
+        </div>
+      {% endif %}
+    </div>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  {% endif %}
+
+  {% if filters.gateway_agreement_id %}
+  <div class="govuk-body">
+    <span>Filtered by gateway agreement ID </span>
+    <span><strong class="govuk-tag">{{ filters.gateway_agreement_id }}</strong></span>
+  </div>
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  {% endif %}
+
```

Comment:
I don't think there's a property to be filtered called gateway agreement id, the equivalent on transactions is for the PSP identifier for payments which I don't think applies here

```suggestion
```

Reply:
Thanks